modules/mesh-layers/src/scenegraph-layer/scenegraph-layer-vertex.glsl.js

```diff
@@ -9,7 +9,7 @@ export default `\
 
 // Instance attributes
 _attribute vec3 instancePositions;
-_attribute vec2 instancePositions64xy;
+_attribute vec2 instancePositions64xyLow;
```

Comment:
This would seem like a good time to drop the `xy` part in the name, and just call this `instancePositions64` and `instancePositions64low`. We could reference them `instancePositions64low.xy` to avoid changing the project function signature at this time.

Better yet, use a with a "generic naming scheme for "component attributes" that also works for matrices etc:

`instancePositions64_1`, `instancePositions64_2`

Reply:
Here we're only renaming two attributes in `SimpleMeshLayer` and `ScenegraphLayer`; if we change the naming convention we would need to change the shaders of all core layers. I'd push it off to the next major release (when we also change the low part attributes to vec3).